frontend/src/components/form/EditableDropdown.vue

```diff
@@ -34,21 +36,29 @@ const { errorMessage, value } = useField<string>(toRef(props, 'name'));
 
 <template>
   <div class="field col">
-    <label :for="name">{{ label }}</label>
+    <label
+      :class="{ 'font-bold': bold }"
+      :for="name"
+    >
+      {{ label }}
+    </label>
     <Dropdown
       v-model.trim="value"
       editable
       :aria-describedby="`${name}-help`"
       :name="name"
       :placeholder="placeholder"
-      :class="'w-full ' + { 'p-invalid': errorMessage }"
+      class="w-full"
+      :class="{ 'p-invalid': errorMessage }"
       :disabled="disabled"
       :options="props.options"
       :option-label="(option) => props.getOptionLabel(option)"
       @input="(e: IInputEvent) => emit('onInput', e)"
       @change="(e: DropdownChangeEvent) => emit('onChange', e)"
     />
     <small :id="`${name}-help`">{{ helpText }}</small>
-    <ErrorMessage :name="name" />
+    <div>
+      <ErrorMessage :name="name" />
+    </div>
```

Comment:
Is the new div wrapper addressing some kind of strange spacing/layout issue? Usually we try to avoid nesting more DOM elements whenever possible.